backend/api/Database/Models/MissionTask.cs

```diff
@@ -152,17 +153,14 @@ or TaskStatus.Failed
 
         public DateTime? EndTime { get; private set; }
 
-        public IList<Inspection> Inspections { get; set; }
+        public Inspection? Inspection { get; set; }
 
         public void UpdateWithIsarInfo(IsarTask isarTask)
         {
             UpdateStatus(isarTask.TaskStatus);
-            foreach (var inspection in Inspections)
+            if (isarTask.TaskType != IsarTaskType.ReturnToHome && isarTask.TaskType != IsarTaskType.Localize && isarTask.TaskType != IsarTaskType.MoveArm)
```

Comment:
```suggestion
            if (![IsarTaskType.ReturnToHome, IsarTaskType.Localize, IsarTaskType.MoveArm].contains(isarTask.TaskType))
```
Subjective change, feel free to ignore

Reply:
This didn't work - There is no target type for the collection expression.[CS9176]